soda/core/tests/data_source/test_distribution_check.py

```diff
@@ -79,7 +79,7 @@ def test_distribution_sql(scanner: Scanner, mock_file_system, table, expectation
         f"{user_home_dir}/the_distribution_check_file.yml": dedent(
             f"""
                 checks for {table_name}:
-                  - distribution_difference(size, my_happy_ml_model_distribution) >= 0.05:
+                  - distribution_difference(size) >= 0.05:
```

Comment:
shouldn't we be testing for a name to be parsed? Since this PR is about allowing DROs to be referred to by name, then I think it would make sense.

Reply:
Passing the DRO name is still optional (the changes that I made in this PR are backwards-compatible). The reason I removed the DRO name in this test is because the distribution reference file that is used does not have a DRO name (so passing `my_happy_ml_model_distribution` as an argument does not make sense). 

That being said, I think you are right that we should test that the DRO name is parsed correctly. I will create a new test that uses a distribution reference file with a named DRO and add it to this PR.